src/compiler/config/test/validate-output-dist.spec.ts

```diff
@@ -30,14 +30,15 @@ describe('validateDistOutputTarget', () => {
         empty: false,
         esmLoaderPath: path.join(rootDir, 'my-dist', 'loader'),
         type: 'dist',
+        polyfills: false,
         typesDir: path.join(rootDir, 'my-dist', 'types'),
       },
       {
         esmDir: path.join(rootDir, 'my-dist', 'my-build', 'testing'),
         empty: false,
         isBrowserBuild: true,
         legacyLoaderFile: path.join(rootDir, 'my-dist', 'my-build', 'testing.js'),
-        polyfills: true,
+        polyfills: false,
```

Comment:
🤔 Do you know what was causing this value to default to `true` before?

Reply:
yes, and I meant to ask about this! It is this bit of code in `validate-output-dist.ts`:

```ts
    // Lazy build for CDN in dist
    outputs.push({
      type: DIST_LAZY,
      esmDir: lazyDir,
      systemDir: config.buildEs5 ? lazyDir : undefined,
      systemLoaderFile: config.buildEs5 ? join(lazyDir, namespace + '.js') : undefined,
      legacyLoaderFile: join(distOutputTarget.buildDir, namespace + '.js'),
      polyfills: distOutputTarget.polyfills !== undefined ? !!distOutputTarget.polyfills : true,
      isBrowserBuild: true,
      empty: distOutputTarget.empty,
    });
```

what do you think about defaulting to `undefined` instead, in `validateOutputTargetDist` ?

Reply:
It's probably best we retain the original behavior (defaulting to `undefined`) here. I think we should do our best here to retain the current/original behavior for now to prevent breaking changes